src/components/InputField/InputField.vue

```diff
@@ -262,10 +281,16 @@ export default {
 
 		&--success {
 			border-color: var(--color-success) !important; //Override hover border color
+			&:focus {
+				box-shadow: 0 0 5px rgba(81, 203, 238, 1);
+			}
 		}
 
 		&--error {
 			border-color: var(--color-error) !important; //Override hover border color
+			&:focus {
+				box-shadow: 0 0 5px rgba(81, 203, 238, 1);
+			}
```

Comment:
About this, could we actually not allow to have focus and success and error borders at the same time? 
I think that success and error borders should be toggled on only when the component is being actively used. So there's no need for the focus state on top of it.